jobs/JWAVE_POST_SBS

```diff
@@ -63,7 +63,7 @@ export EXECwave=${EXECwave:-$HOMEwave/exec}
 export FIXwave=${FIXwave:-$HOMEwave/fix/fix_wave_${NET}}
 export PARMwave=${PARMwave:-$HOMEwave/parm/wave}
 export USHwave=${USHwave:-$HOMEwave/ush}
-export EXECcode=${EXECcode:-$CODEwave/exec}
+export EXECcode=${EXECcode:-$HOMEgfs/exec}
```

Comment:
Please add  export RUN_ENVIR=${RUN_ENVIR:-"nco"}  to the top of all WAVE JJOBS.  Please refer to other JJOBS to see how this is applied to differentiate  EMC parallel and NCO operational environment.   For operation we must have the following lines in all JJOBS

if [ $RUN_ENVIR = "nco" ]; then
    export ROTDIR=${COMROOT:?}/$NET/$envir
fi

Reply:
Line 66:  again what is the difference between EXECcode and EXECwave ?

Reply:
Line 82:  see my previous comment on the use of wavelog

Reply:
lines 85-86:  please define wavempexec and wave_mpmd in ./env/* files.

Reply:
Please add "exit 0" at the end of this script.  Please revise lines 112-115

Reply:
export RUN_ENVIR=${RUN_ENVIR:-"nco"} was added to all wave J-jobs. Other issues were addressed (see previous answers to comments)